### PR TITLE
2. Remove character-PremsaiKodi.cpp

### PR DESCRIPTION
**Pull request by PremsaiKodi**
Remove character in CPP

### CODE REVIEW

**Comment by PremsaiKodi** [comment]
Remove character

**Review by shumbul — CHANGES_REQUESTED** [review]
Great work, please add these changes, will merge

1. Please add time and space complexities as well
2. Rename the file to 2. Remove character-PremsaiKodi.cpp so it does not overwrite existing file
3. Do explore if using unordered_map would be better or map data strcuture based on time complexities

File: 2. Remove character.cpp [hunk 1]
```diff
@@ -4,13 +4,21 @@ class Solution {
...
+            }
+        }
+        return string1;
     }
 };
 
 /*
-Time complexity: O()
-Space complexity: O()
+Time complexity: O(m+n)
```
Comment (PremsaiKodi) [inline 2. Remove character.cpp:20]: m, n are the lengths of string1 and string2 respectively.

**Comment by shumbul** [comment]
@PremsaiKodi file renaming is pending, there is merge conflict rn. You renamed the PR comment. 
Update file name to **2. Remove character-PremsaiKodi.cpp**

**Review by shumbul — APPROVED** [review]

**Pull request merged**